### PR TITLE
D8CORE-1117 Switch to React Paragraphs

### PR DESCRIPTION
**Pull request by pookmish**
# Actually! READY FOR REVIEW

# Summary
- Switch paragraphs field to react paragraphs

# Things to note
- Styling will need some adjustments. because containers and classes are different and [stanford_basic](https://github.com/SU-SWS/stanford_basic/blob/8.x-4.x/stanford_basic.theme#L95) is not applying expected classes right now.
- Tests need to be rebuilt. I'm starting that process now to add some drag and drop behat definitions.
- https://github.com/SU-SWS/acsf-cardinalsites/pull/57

# Need Review By (Date)
- soon

# Urgency
- high

# Steps to Test
1. checkout this branch.
1. either install a new site or run `blt drupal:update` or run `drush updb -y; drush cim -y`
1. edit/create a basic page
1. have fun with the react paragraphs.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by sherakama** [comment]
Hot damn I am excited about this one. I'll hopefully start review soon.

**Comment by sherakama** [comment]
Good news, I got this installed today. I am that much closer to being able to read through it.

**Comment by pookmish** [comment]
Bad news, i'm not much closer to getting tests built.
/blame meetings

File: stanford_profile.post_update.php [hunk 1]
```diff
@@ -22,3 +24,74 @@ function stanford_profile_post_update_8001() {
     }
   }
 }
+
+/**
+ * Change paragraphs fields into react paragraph fields.
+ */
+function stanford_profile_post_update_8003() {
```
Comment (sherakama) [inline stanford_profile.post_update.php]: Nice work. This does the job and no errors in the console or db log after. 👍

File: composer.json [hunk 2]
```diff
@@ -9,6 +9,10 @@
             "type": "composer",
             "url": "https://packages.drupal.org/8"
         },
+        {
+            "type": "vcs",
+            "url": "git@github.com:pookmish/react_paragraphs.git"
```
Comment (sherakama) [inline composer.json]: Where do you want feedback for this guy?

File: composer.json [hunk 3]
```diff
@@ -79,6 +79,7 @@
         "drupal/menu_link_weight": "^1.0-beta3",
         "drupal/menu_position": "~1.0@alpha",
         "drupal/metatag": "~1.3",
+        "drupal/paragraphs": "dev-1.x#9258f0985211427445604ae932df240d363d414d",
```
Comment (sherakama) [inline composer.json]: Can bump when https://www.drupal.org/project/paragraphs/issues/2832021 is in a stable.

**Comment by pookmish** [comment]
Oh snap

**Pull request merged**